### ISSUE TITLE
False error "Unable to import 'os.path'" etc

### ISSUE THREAD

**Issue by pylint-bot** [PyCQA/pylint#159]
Originally reported by: **Anonymous**

---

I use Py3.2 with PyLint as editor plugin in SynWrite editor. For file

```
import os.path
from time   import perf_counter
```

Pylint gives

```
- Unable to import 'os.path'
- No name 'perf_counter' in module 'time'
```

But it's OK py file. Why.

---
- Bitbucket: https://bitbucket.org/logilab/pylint/issue/159
